var/spack/repos/builtin/packages/arrayfire/package.py

```diff
@@ -14,22 +13,19 @@ class Arrayfire(CMakePackage, CudaPackage):
 
     homepage = "https://arrayfire.org/docs/index.htm"
     git      = "https://github.com/arrayfire/arrayfire.git"
+    maintainers = ['umar456']
 
-    version('master', submodules=True)
+    version('master')
+    version('3.8.1', tag='v3.8.1')
     version('3.7.3', submodules=True, tag='v3.7.3')
     version('3.7.2', submodules=True, tag='v3.7.2')
     version('3.7.0', submodules=True, tag='v3.7.0')
 
-    variant('cuda',   default=False, description='Enable Cuda backend')
-    variant('forge',   default=False, description='Enable graphics library')
+    variant('cuda',   default=False, description='Enable CUDA backend')
+    variant('forge',  default=False, description='Enable graphics library')
     variant('opencl', default=False, description='Enable OpenCL backend')
 
-    depends_on('boost@1.65:')
-
-    # TODO: replace this with an explicit list of components of Boost,
-    # for instance depends_on('boost +filesystem')
-    # See https://github.com/spack/spack/pull/22303 for reference
-    depends_on(Boost.with_default_variants)
+    depends_on('boost@1.75:')
```

Comment:
Are you 100% sure no version of array-fire is compatible with older boost? I know for a fact that Arrayfire builds just fine with older versions of boost. But haven't used it to the extent that I would know if there were runtime issues due to boost incompatibility.

Reply:
Well. I guess @umar456 is a developer of ArrayFire, so he would know for sure.

Reply:
There was an issue with Boost Compute that was fixed in a later version. Checking my logs again it looks like it was fixed in 1.70. I incorrectly set the version to 1.75. My apologies. I will update soon.

You CAN build ArrayFire on even older versions of Boost, but internally it clones Boost Compute from version 1.70. I am not sure if this is desired by spack. I plan on removing internal cloning using FetchContent and using spack in a future change to ArrayFire.

Reply:
>but internally it clones Boost Compute from version 1.70. I am not sure if this is desired by spack.

This is not desirable in spack. It would be ideal to let spack handle dependencies like boost.